demo_visualizations.py

```
@@ -0,0 +1,432 @@
+        print("⚠️  No data to analyze")
+        sys.exit(0)
+    
+    analyzer = StatisticalAnalyzer(df)
+    report = analyzer.get_summary_report()
+    
+    print(f"✓ Analysis complete")
+    print(f"  File types: {report['overview'].get('file_type_distribution', {})}")
+    
+    # Save report
+    analyzer.export_report("exports/analysis_report.json")
+    print(f"✓ Report saved to exports/analysis_report.json")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)